README.md

```diff
@@ -1,12 +1,75 @@
 celvet
 ======
 
+**celvet** is a multitool for linting CRDs (`CustomResourceDefinition`) in
+Kubernetes. `celvet` also provides particular insight into [validation rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules),
+checking that they do not exceed the resource constraints provided by
+Kubernetes' API server. The tool is deliberately made to be useful both in an
+ad-hoc fashion by developers in addition to being part of a CI/CD pipeline, so
+`celvet` supports output in JSON in addition to machine-readable text.
+
+What does celvet catch?
+-----------------------
+
+* **CRD parsing errors** - celvet will early exit if it finds that a CRD
+    contains invalid YAML/JSON.
```

Comment:
There's a big difference between validating that YAML/JSON is valid and that the YAML/JSON is a valid CRD.  Does this do the later?

Reply:
Not at the moment - validation rules are linted, and the YAML/JSON itself is parsed of course, but that's not enough to claim that CRDs themselves are validated in my opinion.

Reply:
Okay, that's what I understood as well. The phrase "CRD parsing errors" doesn't feel entirely accurate here.

Are you tracking the lack of CRD validation support as an issue?

Reply:
There's brief mention of it in the (admittedly non-public at the moment) design document, but I've added #11 to track it and reworded the line to better indicate that just the syntax of the YAML/JSON is validated as opposed to the CRD proper.